pkg/container/client.go

```diff
@@ -11,6 +11,8 @@ import (
 	"strings"
 	"time"
 
+        "unsafe"
```

Comment:
Please format code with `goimports` and run `make lint` to check if it properly formatted